source/contracts/reporting/Reporting.sol

```diff
@@ -19,8 +19,8 @@ library Reporting {
     uint256 private constant TARGET_INVALID_MARKETS_DIVISOR = 100; // 1% of markets are expected to be invalid
     uint256 private constant TARGET_INCORRECT_DESIGNATED_REPORT_MARKETS_DIVISOR = 100; // 1% of markets are expected to have an incorrect designate report
     uint256 private constant TARGET_DESIGNATED_REPORT_NO_SHOWS_DIVISOR = 100; // 1% of markets are expected to have an incorrect designate report
-    uint256 private constant TARGET_REP_MARKET_CAP_MULTIPLIER = 15; // We multiply and divide by constants since we want to multiply by a fractional amount (7.5)
-    uint256 private constant TARGET_REP_MARKET_CAP_DIVISOR = 2;
+    uint256 private constant TARGET_REP_MARKET_CAP_MULTIPLIER = 3; // We multiply and divide by constants since we may want to multiply by a fractional amount
+    uint256 private constant TARGET_REP_MARKET_CAP_DIVISOR = 1;
```

Comment:
How were these numbers arrived at?

Reply:
Game Theory discussion / will be explained in updates to the whitepaper. This is the minimum target OI with the new Use or Lose system implemented. We want the lowest target OI possible to minimize fees

Reply:
I don't remember actually coming to consensus on 3?  I know it was thrown around as a straw man number, but I do think we should make sure to think through what the right number is before v2.

Reply:
Of course. Will re-review after the whitepaper is updated